lib/view_component/base.rb

```diff
@@ -5,6 +5,41 @@
 require "view_component/previewable"
 
 module ViewComponent
+  class CollectionBase
```

Comment:
One more thing. Better extract this class into a different file.

Reply:
Done.